### DISCUSSION TITLE
Github Education as a student while I'm a teacher

### DISCUSSION THREAD

**Discussion by agila5** [community/community#120051] [GitHub Education]
### Hi everyone,

I created this discussion since my request to join GitHub Education was recently approved but, as far as I can understand, I got signed in as a student while I'm a teacher. Is it possible to switch the role (after giving appropriate credentials that I'm a teacher)? If so, how? 

Similar problem: https://github.com/orgs/community/discussions/51019

**Comment by samus-aran** [comment]
Hi @agila5 👋 

We've revoked your current verified status. You will have to re-submit as a Teacher. 👍
